### ISSUE TITLE
expor shows the input cell code

### ISSUE THREAD

**Issue by prtome** [m-rossi/jupyter-docx-bundler#93]
**Describe the bug**
I have just updated to nbconvert 7.2.6. I have run my export as usual : 
   python3 -m jupyter nbconvert --execute --to=docx alpha.ipynb --output alpha.docx

The problem is that now all the code cells are showing in the word document.
The metadata is correctly telling it not to do that.

**Desktop (please complete the following information):**
- OS: MACOS 12.5.1
- Browser [e.g. chrome, safari] chrome and safari
- Versions of Python packages, find out with `pip list`: all updated  today dec 13, 2022
- Pandoc version `pandoc --version` pandoc 2.17.1.1

pip list:
Package                           Version
--------------------------------- -----------
anyio                             3.6.2
appdirs                           1.4.4
appnope                           0.1.3
argon2-cffi                       21.3.0
argon2-cffi-bindings              21.2.0
arrow                             1.2.3
asttokens                         2.2.1
attrs                             22.1.0
autopep8                          2.0.0
Babel                             2.11.0
backcall                          0.2.0
beautifulsoup4                    4.11.1
black                             22.12.0
bleach                            5.0.1
bokeh                             3.0.3
certifi                           2022.12.7
cffi                              1.15.1
charset-normalizer                2.1.1
click                             8.1.3
colorlover                        0.3.0
comm                              0.1.2
contourpy                         1.0.6
cufflinks                         0.17.3
cycler                            0.11.0
debugpy                           1.6.4
decorator                         5.1.1
defusedxml                        0.7.1
entrypoints                       0.4
executing                         1.2.0
fastjsonschema                    2.16.2
fonttools                         4.38.0
fqdn                              1.5.1
idna                              3.4
import-ipynb                      0.1.4
importlib-metadata                5.1.0
ipykernel                         6.19.2
ipython                           8.7.0
ipython-genutils                  0.2.0
ipywidgets                        8.0.3
isoduration                       20.11.0
jedi                              0.18.2
Jinja2                            3.1.2
joblib                            1.2.0
json5                             0.9.10
jsonpointer                       2.3
jsonschema                        4.17.3
jupyter                           1.0.0
jupyter_client                    7.4.8
jupyter-console                   6.4.4
jupyter-contrib-core              0.4.2
jupyter-contrib-nbextensions      0.7.0
jupyter_core                      5.1.0
jupyter-docx-bundler              0.3.4
jupyter-events                    0.5.0
jupyter-highlight-selected-word   0.2.0
jupyter-latex-envs                1.4.6
jupyter-nbextensions-configurator 0.6.1
jupyter_server                    2.0.1
jupyter_server_terminals          0.4.2
jupyterlab                        3.5.1
jupyterlab-pygments               0.2.2
jupyterlab_server                 2.16.5
jupyterlab-widgets                3.0.4
kaleido                           0.2.1
keras                             2.11.0
kiwisolver                        1.4.4
lxml                              4.9.1
MarkupSafe                        2.1.1
matplotlib                        3.6.2
matplotlib-inline                 0.1.6
mistune                           2.0.4
mlxtend                           0.21.0
mypy-extensions                   0.4.3
nbclassic                         0.4.8
nbclient                          0.7.2
nbconvert                         7.2.6
nbformat                          5.7.0
nest-asyncio                      1.5.6
notebook                          6.5.2
notebook_shim                     0.2.2
numpy                             1.23.5
ordered-set                       4.1.0
packaging                         22.0
pandas                            1.5.2
pandoc                            2.3
pandocfilters                     1.5.0
parso                             0.8.3
pathspec                          0.10.3
pexpect                           4.8.0
pickleshare                       0.7.5
Pillow                            9.3.0
pip                               22.3.1
platformdirs                      2.6.0
plotly                            5.11.0
plumbum                           1.8.0
ply                               3.11
prometheus-client                 0.15.0
prompt-toolkit                    3.0.36
psutil                            5.9.4
ptyprocess                        0.7.0
pure-eval                         0.2.2
pycodestyle                       2.10.0
pycparser                         2.21
pyee                              9.0.4
Pygments                          2.13.0
PyLaTeX                           1.4.1
pypandoc                          1.10
pyparsing                         3.0.9
pyppeteer                         1.0.2
pyrsistent                        0.19.2
python-dateutil                   2.8.2
python-json-logger                2.0.4
pytz                              2022.6
PyYAML                            6.0
pyzmq                             24.0.1
qtconsole                         5.4.0
QtPy                              2.3.0
requests                          2.28.1
rfc3339-validator                 0.1.4
rfc3986-validator                 0.1.1
scikit-learn                      1.2.0
scikit-multilearn                 0.2.0
scipy                             1.9.3
seaborn                           0.12.1
Send2Trash                        1.8.0
setuptools                        65.6.3
six                               1.16.0
sniffio                           1.3.0
soupsieve                         2.3.2.post1
stack-data                        0.6.2
tabulate                          0.9.0
tenacity                          8.1.0
terminado                         0.17.1
testpath                          0.6.0
threadpoolctl                     3.1.0
tinycss2                          1.2.1
toml                              0.10.2
tomli                             2.0.1
tornado                           6.2
tqdm                              4.64.1
traitlets                         5.7.1
typing_extensions                 4.4.0
uri-template                      1.2.0
urllib3                           1.26.13
voila                             0.4.0
wcwidth                           0.2.5
webcolors                         1.12
webencodings                      0.5.1
websocket-client                  1.4.2
websockets                        10.4
wheel                             0.38.4
widgetsnbextension                4.0.4
xyzservices                       2022.9.0
yellowbrick                       1.5
zipp                              3.11.0

**Comment by m-rossi** [comment]
Thanks for reporting @prtome. Just a minor follow-up question, am I right you use the [hiding all inputs](https://github.com/m-rossi/jupyter-docx-bundler#hiding-all-inputs)-approach?

**Comment by prtome** [comment]
yes. I have that in all my docs, this one was exporting perfectly before my updates.
So I am sure something is happening with some library new version

**Comment by m-rossi** [comment]
@prtome just released [0.3.5](https://github.com/m-rossi/jupyter-docx-bundler/releases/tag/0.3.5) on PyPI and also will be soon on conda-forge. Please test if it fixes your issue and report back.

**Comment by prtome** [comment]
Yes ! installed and problem solved ! Thanks
